src/components/InputUnit.vue

```diff
@@ -51,12 +52,27 @@ export default class InputUnit extends Vue {
   get displayValue() {
     let convertedValue = this.toDisplay(this.value);
 
+    if (convertedValue === null || convertedValue === undefined)
```

Comment:
```suggestion
    if (convertedValue == null)
```

should work and would be consistent with the existing code style. `== null` checks for both `null` or `undefined` at the same time.